ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml

```diff
@@ -95,10 +95,9 @@ tests:
     cluster_profile: aws-virtualization
     env:
       BASE_DOMAIN: cnv-ci.syseng.devcluster.openshift.com
-      CNV_PRERELEASE_VERSION: "4.13"
+      CNV_PRERELEASE_VERSION: "4.14"
       COMPUTE_NODE_REPLICAS: "1"
       COMPUTE_NODE_TYPE: c5n.metal
-      CONTROL_PLANE_INSTANCE_TYPE: m4.xlarge
```

Comment:
Default instace type is fine now ?

Reply:
yep, we actually set the instance type correctly here, https://github.com/openshift/release/blob/0aba461ea704e9a5b94a120537ff53dfa235b0ef/ci-operator/step-registry/hypershift/kubevirt/e2e/capk/hypershift-kubevirt-e2e-capk-workflow.yaml#L13

I believe this line was overriding the default we wanted to use, but since this job executes so infrequently we just didn't notice yet.